chains/solana/contracts/programs/ccip-router/src/instructions/v1/mod.rs

```diff
@@ -4,6 +4,7 @@
 pub mod admin;
 pub mod offramp;
 pub mod onramp;
+pub mod price_math;
```

Comment:
I've made this module public as some of the utilities were necessary to call from `onramp`, not just `fee_quoter`, plus they seemed to be mostly utility functions. LMK if that's okay with your refactor @toblich